### PR TITLE
high contrast bug fixes

### PR DESCRIPTION
**Pull request by Anush2303**
## New Behavior

Fixed bugs:
1. https://uifabric.visualstudio.com/iss/_workitems/edit/11592
2. https://uifabric.visualstudio.com/iss/_workitems/edit/11589
3. https://uifabric.visualstudio.com/iss/_workitems/edit/11588
4. https://uifabric.visualstudio.com/iss/_workitems/edit/11574
5. https://uifabric.visualstudio.com/iss/_workitems/edit/11632

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.styles.ts [hunk 1]
```diff
@@ -164,4 +165,22 @@ export const styles: ElementStyles = css`
     ${typographyBody1StrongStyles}
     color: ${colorNeutralForeground1};
   }
-`;
+  @media (forced-colors: active) {
+    .bar-label {
```
Comment (krkshitij) [inline packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.styles.ts:169]: Can these styles be included within `forcedColorsStylesheetBehavior`?

Comment (Anush2303) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.styles.ts:169]: Done

Comment (krkshitij) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.styles.ts]: The media query is redundant as it is already managed by `forcedColorsStylesheetBehavior`.
See this: https://github.com/microsoft/fluentui/blob/85a3aa172ee82f8477eeb4f2ae5a85f52920b82a/packages/web-components/src/utils/behaviors/match-media-stylesheet-behavior.ts#L171

Comment (Anush2303) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.styles.ts]: Okay

**Review by krkshitij — APPROVED** [review]

**Pull request merged**